content/cucumber/reporting.md

```diff
@@ -9,8 +11,18 @@ what scenarios have passed or failed.
 Some plugins are built-in, others have to be installed separately. You can also
 build your own.
 
+This page documents built-in formatter plugins as well common third-party plugins.
+Available plugins may vary per programming language.
+
 # Built-in reporter plugins
 
+There are several reporter plugins built into Cucumber:
+
+* progress
+* pretty
+* html
```

Comment:
could these have teletype styling, like `progress`, so that they stand out next to the plaintext "etc."?

Reply:
Sure, will do right away.

Reply:
LGTM!